tests/views/test_path.py

```diff
@@ -47,4 +47,4 @@ def test_repo_name_without_manifest_or_tags_or_blobs(self):
         response = self.test_client.get('/v2/foo/test')
 
         self.assertEqual(response.status_code, 404)
-        self.assertTrue(response.headers['Content-Type'].startswith('text/html'))
+        self.assertTrue(response.headers['Content-Type'].startswith('application/json'))
```

Comment:
Can you strengthen the assertions here as well? In addition to asserting that they are the correct type, you should make sure the JSON output is correct since that is the goal of the pull request.